lib/ssl/doc/src/ssl.xml

```diff
@@ -1610,12 +1624,14 @@ fun(srp, Username :: binary(), UserState :: term()) ->
     
     <func>
       <name name="clear_pem_cache" arity="0" since="OTP 17.5"/>
-      <fsummary> Clears the pem cache</fsummary>
+      <fsummary>Clears the pem cache</fsummary>
       <desc><p>PEM files, used by ssl API-functions, are cached. The
       cache is regularly checked to see if any cache entries should be
       invalidated, however this function provides a way to
       unconditionally clear the whole cache.
       </p>
+      <warning><p>Calling this function ensures that cache becomes empty and
+      previously cached data will not be used for following connections.</p></warning>
```

Comment:
I don't think this is necessary, at least not as a warning. It sort of says what the previous paragraph already says, so boils down to a warning that it will do what you want :wink:

Reply:
You're right. I tried reformulating it. Is it better now?